flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/SqlCommandParserTest.java

```diff
@@ -76,6 +76,10 @@ public void testCommands() {
 		testInvalidSqlCommand("source"); // missing path
 		testValidSqlCommand("USE CATALOG default", new SqlCommandCall(SqlCommand.USE_CATALOG, new String[]{"default"}));
 		testValidSqlCommand("use default", new SqlCommandCall(SqlCommand.USE, new String[] {"default"}));
+		testValidSqlCommand("create Table hello(a int)", new SqlCommandCall(SqlCommand.CREATE_TABLE, new String[]{"create Table hello(a int)"}));
+		testValidSqlCommand("  CREATE TABLE hello(a int)", new SqlCommandCall(SqlCommand.CREATE_TABLE, new String[]{"CREATE TABLE hello(a int)"}));
+		testInvalidSqlCommand("CREATE tables");
```

Comment:
Add tests for statements:

1. `create table t(a int, b timestamp(6)) with ('k1' = 'v1', 'k2' = 'v2')`
2. `create table t(a int, b timestamp(6), c as a + 1) with ('k1' = 'v1', 'k2' = 'v2')`
3. `create table t(a int, b timestamp(3), watermark for b as my_udf(b, 1) - INTERVAL '5' second) with ('k1' = 'v1', 'k2' = 'v2')`

Reply:
For each positive tests we can make some negative ones.